lib/timeline/component/css/item.css

```diff
@@ -90,6 +90,16 @@
   padding: 5px;
 }
 
+.vis-item .vis-onUpdateTime-tooltip {
```

Comment:
Please add `text-align: center` and `white-space: nowrap`, make it autosize and remove the double `background`:
```css
.vis-item .vis-onUpdateTime-tooltip {
    position: absolute;
    color: white;
    width: auto;
    background: #4f81bd;
    padding: 5px;
    border-radius: 1px;
    text-align: center;
    white-space: nowrap;
}